transformer_engine/paddle/csrc/custom_ops.cu

```diff
@@ -8,8 +8,9 @@
 #include <map>
 #include <vector>
 
+#include "../common.h"
```

Comment:
The ".." is at transformer_engine/paddle, but there is no common.h.

Reply:
I may have missed this during the rebase process. I will restored to `#include "common/common.h"`. Thank you for pointing this out.